### PR TITLE
Sld wide svg fix

### PR DESCRIPTION
**Pull request by CBiasuzzi**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
bug fix

**What is the current behavior?** *(You can also link to an open issue here)*
display logic in the SLD viewer when the svg is too big, to display the left corner: fixes the zoom computation

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/single-line-diagram-viewer.ts [hunk 1]
```diff
@@ -233,8 +233,8 @@ export class SingleLineDiagramViewer {
             //so that panzoom's margins still work correctly.
             //I am not sure the offsetX and offsetY thing is correct. It seems
             //to help. When someone finds a big problem, then we can fix it.
-            const newLvlX = svgWidth / maxWidth / 2;
-            const newLvlY = svgHeight / maxHeight / 2;
+            const newLvlX = svgWidth / (maxWidth + 200);
```
Comment (etiennehomer) [inline src/single-line-diagram-viewer.ts:236]: The formula works fine for some large SLD but not some others. 
By example, the rendering is incorrect for VLEJUP7 of a Recollement...

Comment (CBiasuzzi) [reply src/single-line-diagram-viewer.ts]: Thanks for the feedback.
I made a couple of changes, then tried with these: VLEJUP7, VLEJU, VOUGL, VOID, ARGOE.
Now, to me, it seem indistinguishable from the rendering done by the current gridstudy-app.
Could you please take another look?

**Review by etiennehomer — APPROVED** [review]

File: src/single-line-diagram-viewer.ts [hunk 2]
```diff
@@ -233,22 +233,22 @@ export class SingleLineDiagramViewer {
             //so that panzoom's margins still work correctly.
             //I am not sure the offsetX and offsetY thing is correct. It seems
             //to help. When someone finds a big problem, then we can fix it.
-            const newLvlX = svgWidth / maxWidth / 2;
-            const newLvlY = svgHeight / maxHeight / 2;
+            const newLvlX = svgWidth / maxWidth;
+            const newLvlY = svgHeight / maxHeight;
 
-            const xOrigin = dimensions.viewbox.x - 20;
```
[the remark this answers is not in the document]
Comment (CBiasuzzi) [reply src/single-line-diagram-viewer.ts:239]: the method getDimensionsFromSvg already takes care of the original -20.

[the remark this answers is not in the document]
Comment (etiennehomer) [reply src/single-line-diagram-viewer.ts:239]: Yes, perfect

**Comment by etiennehomer** [comment]
Let's merge and I release a new version ?

**Comment by CBiasuzzi** [comment]
> Let's merge and I release a new version ?

yes, please!

**Pull request merged**